src/components/policy/test/policy_manager_impl_test.cc

```diff
@@ -253,6 +254,22 @@ TEST_F(PolicyManagerImplTest, LoadPT_SetPT_PTIsLoaded) {
   app_policies["1234"]["steal_focus"] = Json::Value(true);
   app_policies["1234"]["certificate"] = Json::Value("sign");
 
+  app_policies["device"] = Json::Value(Json::objectValue);
+  app_policies["device"]["keep_context"] = Json::Value(false);
```

Comment:
@JackLivio Sorry I have mentioned the ticket #517. These value should be omitted in in Open source due to current Ford's requirements